### PR TITLE
[MMA] Vendor implementations area

### PR DESCRIPTION
**Pull request by rabi-abdel**
Fixes https://github.com/cntt-n/CNTT/issues/1304

### CODE REVIEW

File: doc/ven_impl/ven_template/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# CNTT Vendor Implementation - VI
+
+## Vendor Name
+Vendor XX
```
Comment (pgoyal01) [inline doc/ven_impl/ven_template/README.md]: Should the Vendor information include Vendor Contacts

File: doc/ven_impl/ven_template/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Target Reference Architecture :
+|   Type          | Release  | Notes
+|--------------------------------|--------------------------------|-------------------------------|
+| OpenStack (RA-1) |  v3.0   |   |
```
Comment (pgoyal01) [inline doc/ven_impl/ven_template/README.md]: Are the releases version #s or the names we have been assigning. Personally I prefer Version numbers.

File: doc/ven_impl/ven_template/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Target Reference Conformance :
+|   Type          | Release  | Notes
+|--------------------------------|--------------------------------|-------------------------------|
+| OpenStack (RC-1) |  v3.0   |   |
+
+## VI Release Information
```
Comment (pgoyal01) [inline doc/ven_impl/ven_template/README.md]: Is it sufficient to get version numbers or do we need more details about the Vendor Implementation

File: doc/ven_impl/ven_template/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+**Release Date: _17th April 2020_**
+
+## Version History
+
+| Version | Date | Note
+| --- | --- | --- |
+| 1.0-alpha | TBD | TBD|
+
+## Content
+* [Cookbook](./cookbook.md)
```
Comment (pgoyal01) [inline doc/ven_impl/ven_template/README.md]: will every Vendor have their own cookbooks? If yes, then the cookbooks need Vendor Information

File: doc/ven_impl/ven_template/cookbook.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# CNTT - VI Cookbook
```
Comment (pgoyal01) [inline doc/ven_impl/ven_template/cookbook.md]: If every Vendor has their own cookbook then need Vendor information (Vendor Name, Vendor Contact, Cookbook version, Cookbook date)

File: doc/ven_impl/ven_template/report.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
+# CNTT - VI Report
```
Comment (pgoyal01) [inline doc/ven_impl/ven_template/report.md]: need Vendor information (Vendor Name, Vendor Contact, Report version, Report date)

File: doc/ven_impl/ven_template/cookbook.md (hunk 5 above)
Comment (rabiabdel) [reply doc/ven_impl/ven_template/cookbook.md]: @pgoyal01 great input. Each vendor has a folder and hence all vendor information are on README.md in the root of their folder (currently only the name)

File: doc/ven_impl/ven_template/report.md (hunk 6 above)
Comment (rabiabdel) [reply doc/ven_impl/ven_template/report.md]: @pgoyal01 same as above.

File: doc/ven_impl/ven_template/report.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+> Step by step instructions to run RCx against the implementation.
+> Should adopt similar process as written in RC-1
+
+<a name="4"></a>
+## 4 Results
+
+> Detailed results of the test outputs.
+> if possible, pointers and links to public results (such as those of OVP)
+> Links to LFN badges, etc.
```
Comment (pgoyal01) [inline doc/ven_impl/ven_template/report.md]: Sorry if I misunderstood, the approvals are in the Vendor Implementation report (has links to these Test Results) so how will there e badges in the Test Report? Also, are there badges during the "trials"?

File: doc/ven_impl/ven_template/README.md (hunk 1 above)
Comment (rabi-abdel) [reply doc/ven_impl/ven_template/README.md]: added in https://github.com/cntt-n/CNTT/pull/1305/commits/64eff759468018eb504868f6f57964dd1658338e

File: doc/ven_impl/ven_template/README.md (hunk 2 above)
Comment (rabi-abdel) [reply doc/ven_impl/ven_template/README.md]: Yes they are version numbers

File: doc/ven_impl/ven_template/README.md (hunk 3 above)
Comment (rabi-abdel) [reply doc/ven_impl/ven_template/README.md]: more details will be in the cookbook, this is release details of the VI in CNTT. NFVI version that relates to a product will be in the cookbook itself.

File: doc/ven_impl/ven_template/README.md (hunk 4 above)
Comment (rabi-abdel) [reply doc/ven_impl/ven_template/README.md]: Yes, but that information will be stored once in README.md insider vendor own folder.

File: doc/ven_impl/ven_template/report.md (hunk 7 above)
Comment (rabi-abdel) [reply doc/ven_impl/ven_template/report.md]: same template will be used for trials or for simply a vendor to provide their own conformant implementations. in either case, those has to be signed off by CNTT members. the way we sign them off can be by relying on the badge as well as someone to validate the instructions and nothing is broken.

**Pull request merged**